x-pack/legacy/plugins/siem/public/pages/detection_engine/translations.ts

```diff
@@ -51,3 +51,34 @@ export const EMPTY_ACTION_SECONDARY = i18n.translate(
     defaultMessage: 'Go to documentation',
   }
 );
+
+export const NO_INDEX_TITLE = i18n.translate('xpack.siem.detectionEngine.noIndexTitle', {
+  defaultMessage: 'Let’s setup your detection engine',
+});
+
+export const NO_INDEX_MSG_BODY = i18n.translate('xpack.siem.detectionEngine.noIndexMsgBody', {
+  defaultMessage:
+    'To begin using the detection engine, a user with permission must first access this page to create the necessary Elasticsearch index. Doing so will automatically generate the required index and allow users to begin generating signals from rules. As you currently don’t have these permissions, please contact your administrator for further assistance.',
```

Comment:
Should this be more specific?  For example: "Before the SIEM detection engine can be used, a user with `create_index` privilege on the SIEM signals index for this Kibana Space must first access this page. Doing so will cause the required index to be created and will allow users to activate rules that can generate signals. You do not currently have this permission. Please contact your Elastic Stack administrator for further assistance."

Reply:
I'd keep this simple, as users without the required privileges will always need to contact an admin. Suggested text:

To use the detection engine, a user with the required cluster and index privileges must first access this page. For more help, please contact your administrator.